Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go

```diff
@@ -51,39 +51,28 @@ $ kubectl logs --since=1h nginx`
 )
 
 type LogsOptions struct {
-	Client *client.Client
-
-	PodNamespace  string
-	PodName       string
-	ContainerName string
-	Follow        bool
-	Timestamps    bool
-	Previous      bool
-	LimitBytes    int
-	Tail          int
-	SinceTime     *unversioned.Time
-	SinceSeconds  time.Duration
-
-	Out io.Writer
+	Namespace   string
+	ResourceArg string
+	Options     runtime.Object
+
+	Factory *cmdutil.Factory
```

Comment:
is there any way this could have a PodLogsNamespacer, not a factory? Requiring a factory makes it really hard to reuse this from other commands